Solutions/Windows Security Events/Hunting Queries/GroupAddedToPrivlegeGroup.yaml

```diff
@@ -54,4 +53,15 @@ query: |
   GroupAddition
   ) on GroupSid
   | extend timestamp = GroupCreateTime, AccountCustomEntity = Account, HostCustomEntity = Computer
+  | extend NTDomain = tostring(split(AccountCustomEntity,'\\',0)[0]), Name = tostring(split(AccountCustomEntity,'\\',1)[0])
+entityMappings:
+  - entityType: Account
+      - identifier: Name
+        columnName: Name
+      - identifier: NTDomain
+        columnName: NTDomain
+  - entityType: Host
+    fieldMappings:
+      - identifier: FullName
```

Comment:
Change this to specific identifiers versus the special identifier of FullName here.

Reply:
Done